widget/radio_extended_test.go

```diff
@@ -135,59 +137,48 @@ func TestRadio_Extended_Hovered(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			radio := newExtendedRadio(tt.options, nil)
 			radio.Horizontal = tt.isHorizontal
-			render := cache.Renderer(radio).(*radioRenderer)
+			item1 := radio.items[0]
+			render1 := cache.Renderer(item1).(*radioItemRenderer)
+			render2 := cache.Renderer(radio.items[1]).(*radioItemRenderer)
 
-			assert.Equal(t, false, radio.hovered)
-			assert.Equal(t, theme.BackgroundColor(), render.items[0].focusIndicator.FillColor)
-			assert.Equal(t, theme.BackgroundColor(), render.items[1].focusIndicator.FillColor)
+			assert.False(t, item1.hovered)
```

Comment:
Just for clarity of the above, changing the test code here is OK, as it's internal render detail :)